client/gutenberg/editor/style.scss

```diff
@@ -177,9 +177,14 @@ $gutenberg-theme-toggle: #11a0d2;
 	}
 
 	// UNSET CALYPSO DEFAULT STYLES
-	input[type='text'],
-	input[type='search'] {
-		width: auto;
+		input[type='text'],
+		input[type='search'] {
+					// This is overriding the width of the .components-text-control__input elements because
+			 		// .is-section-gutenberg-editor input[type='text'] is more specific, so we add this :not selector in order to
+			 		// don't change the width defined by @wordpress/components
+			 		&:not( .components-text-control__input ) &:not( .editor-format-toolbar__link-container-content ) {
```

Comment:
I don't think chaining `nots` like this is valid CSS, that's the reason why we removed it in previous iteration. You can verify that this fix still works if you remove this line and keep the more specific changes bellow.

Reply:
That's a good catch, thanks! Yes, it seems unnecessary because everything still works as expected for me without it. :)